pkg/importer/dataStream.go

```diff
@@ -693,6 +704,58 @@ func copy(r io.Reader, out string, qemu bool, imageSize string) error {
 	return nil
 }
 
+//Validates wheather there is enough space in PVC for both qcow2 image and converted raw image
+//Assumes it is possible to retrieve info from qcow2 file by means of qemu-img utility
+//Returns failure if either of the following happens
+//1. info cannot be treieved from qcow2 file
+//2. Either ActrualSize or VirtualSize are 0 - not specifed
+//3. ActualSize and VirtualSize together exceed available PVC Space
+func ValidateSpaceConstraint(qcow2Image string, destDir string, qemu bool) (bool, error) {
```

Comment:
The caller of ValidateSpaceConstraint should know if they are in qemu mode or not. I think it makes sense to have them decided based on that flag to call ValidateSpaceConstraint, instead of making this a no-op based on that flag. So I would remove the qemu flag and check in this function and instead do the check in dataStream.go something like
```go
if qemu {
  isEnoughSpace, err := ValidateSpaceConstraint(dest, filepath.Dir(out))
  if err != nil {
    return err
  }
  if !isEnoughSpace {
    return errors.Wrap(err, "Local qcow2 to raw conversion failed - there is not enough space for conversion")
  }
}
```